### PR TITLE
Feature/Docker network

### PR DESCRIPTION
**Pull request by sergiopalacio**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,6 +1,6 @@
-APP_NAME=Laravel
+APP_NAME=sparring-storage
 APP_ENV=local
-APP_KEY=
```
Comment (othercodes) [inline .env.example:3]: This is required? is the example file

**Review by othercodes — CHANGES_REQUESTED** [review]

File: .env.example [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+APP_KEY=base64:eQpZcfvY9Fv/sKi0GFs2VZsCAOT8mXO22NwzJoLhL9M=
```
Comment (othercodes) [inline .env.example:3]: Is this required?? is an example file...

**Review by othercodes — CHANGES_REQUESTED** [review]

File: .env.example (hunk 2 above)
Comment (othercodes) [reply .env.example:3]: Any update on this?

Comment (sergiopalacio) [reply .env.example:3]: Hi I missed that it should not be there

**Review by othercodes — APPROVED** [review]

**Pull request merged**